insights/specs/datasources/candlepin_broker.py

```diff
@@ -56,7 +56,7 @@ def candlepin_broker(broker):
         if content:
             root = ET.fromstring('\n'.join(content))
             # remove namespace before save to avoid urgly search
-            for node in root.getiterator():
+            for node in list(root.iter()):
```

Comment:
@ryan-blakley since `.iter()` was added in python 2.7, what about python 2.6 support?  I wonder if the existing tests are not covering this code?

Reply:
@bfahr I think 2.6 clients use the ET that's in contrib which is the 2.7 version backported I thought.

Reply:
@ryan-blakley that may be the case, thanks for checking.